pygments/lexers/tal.py

```diff
@@ -0,0 +1,58 @@
+"""
+    pygments.lexers.uxntal
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexers for Uxntal
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+import keyword
+
+from pygments.lexer import RegexLexer
+from pygments.token import Comment, Keyword, Name, String, Number, Punctuation, Whitespace, Literal
+
+__all__ = ['TalLexer']
+
+class TalLexer(RegexLexer):
+
+    name = 'Tal'
+    aliases = ['tal', 'uxntal']
+    filenames = ['*.tal']
+    mimetypes = ['text/x-uxntal']
+
+    tokens = {
+        'comment': [
+            (r'(?:(?<=\s)|^)\((?:(?=\s)|$)', Comment.Multiline, '#push'), # nested comments
+            (r'(?:(?<=\s)|^)\)(?:(?=\s)|$)', Comment.Multiline, '#pop'), # nested comments
+            (r'\S+', Comment.Multiline), # comments
+            (r'\s+', Comment.Multiline), # comments
+        ],
+        'root': [
+            (r'\s+', Whitespace), # spaces
+            (r'(?:(?<=\s)|^)\((?:(?=\s)|$)', Comment.Multiline, 'comment'), # comments
+            (r'(?:(?<=\s)|^)(BRK|LIT|INC|POP|DUP|NIP|SWP|OVR|ROT|EQU|NEQ|GTH|LTH|JMP|JCN|JSR|STH|LDZ|STZ|LDR|STR|LDA|STA|DEI|DEO|ADD|SUB|MUL|DIV|AND|ORA|EOR|SFT)2?k?r?(?:(?=\s)|$)', Keyword.Reserved), # instructions
```

Comment:
`words(('BRK', 'LIT', ...), prefix=r'(?<!\S)'`

We have a `words()` function for these kinds of lists, optimizing the regex.

Reply:
I'll take a look at that, thanks!